contracts/Staking.sol

```diff
@@ -306,6 +306,7 @@ contract Staking is IStaking, Initializable, OwnableUpgradeable, Constants {
             indexerLength++;
         } else {
             require(msg.sender == _indexer, 'Only indexer');
+            require(IDisputeManager(settings.getDisputeManager()).disputeIdByIndexer(_indexer) == 0, 'indexer on dispute');
```

Comment:
Would be good to apply the dispute check on `_delegateToIndexer` and `_startUnbond`, so that delegate and undelegate will also be locked